backend/src/main/kotlin/app/ehrenamtskarte/backend/stores/importer/steps/Sanitize.kt

```diff
@@ -1,41 +1,35 @@
 package app.ehrenamtskarte.backend.stores.importer.steps
 
+import app.ehrenamtskarte.backend.common.STATE
+import app.ehrenamtskarte.backend.stores.geocoding.calculateDistanceInKm
+import app.ehrenamtskarte.backend.stores.geocoding.queryFeatures
 import app.ehrenamtskarte.backend.stores.importer.PipelineStep
 import app.ehrenamtskarte.backend.stores.importer.types.AcceptingStore
-import com.fasterxml.jackson.databind.ObjectMapper
-import io.ktor.client.HttpClient
-import io.ktor.client.request.request
-import io.ktor.http.*
 import kotlinx.coroutines.runBlocking
 import org.geojson.Feature
-import org.geojson.FeatureCollection
 import org.geojson.Point
 import org.slf4j.Logger
-import java.io.File
-import kotlin.math.*
 
-const val STATE = "Bayern"
-
-const val DISTANCE_THRESHOLD = 5
+const val DISTANCE_THRESHOLD_IN_KM = 5
```

Comment:
```suggestion
const val MAX_DISTANCE_THRESHOLD_IN_KM = 5
```

Reply:
it's a distance *threshold*, so no `max` needed. Sorry Max...